packages/files-ui/cypress/support/page-objects/homePage.ts

```diff
@@ -8,9 +8,9 @@ export const homePage = {
   uploadFileForm: () => cy.get("[data-cy=upload-file-form] input"),
   moveSelectedButton: () => cy.get("[data-cy=move_selected_file_button]"),
   deleteSelectedButton: () => cy.get("[data-cy=delete_selected_file_button]"),
-  deleteFileDialog: () => cy.get("[data-testid=file-deletion-modal]"),
-  deleteFileCancelButton: () => cy.get(".makeStyles-dashed-97"),
-  deleteFileConfirmButton: () => cy.contains("Confirm"),
+  deleteFileDialog: () => cy.get("[data-testid=modal-container-file-deletion]"),
+  deleteFileCancelButton: () => cy.get("[data-testid=cancel-deletion-button]"),
+  deleteFileConfirmButton: () => cy.get("[data-testid=confirm-deletion-button]"),
   uploadStatusToast: () => cy.get("[data-cy=upload_status_toast_message]", { timeout: 10000 }),
```

Comment:
I see this a little late but for consistency we should use `-` instead of `_` (that's just a suggestion for here but it would need to be adapted in the code too)
```suggestion
  uploadStatusToast: () => cy.get("[data-cy=upload-status-toast-message]", { timeout: 10000 }),
```

Reply:
@Tbaut Oh! That was a legitimate mistake, thanks for catching it 👍